stl_generator/stl_util.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
 from shapely.geometry import Polygon, Point
 
-from geotiff import GeoTIFFS
+from .geotiff import GeoTIFFS
```

Comment:
On Mac this threw an error about not finding the GeoTIFFS module. I was able to resolve by forcing a local import. This seems to work inside the Debian container, does it work for localhost Linux too?